src/index.test.ts

```diff
@@ -650,7 +650,7 @@ test('Should abort previous request', async () => {
     client.updateContext({ userId: '456' }); // abort 2
     await client.updateContext({ userId: '789' });
 
-    expect(abortSpy).toBeCalledTimes(2);
```

Comment:
how are they different?

Reply:
one is deprecated. just a rename in a library - no real difference